docs/ingredients/desert.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Desert Menu
```

Comment:
I'm still not sold that this section is needed.

Reply:
was already removed from the preview in vercel, along with the re structure and naming of certain things